README.md

```diff
@@ -276,11 +276,11 @@ Kinvey CLI is distributed as an NPM package. After you install NPM, run the foll
     
 * `flex job [id]`
 
-   Shows the job status of a deploy/recycle command. If you don't specify an `id`, the command returns the status of the most recent `flex deploy` or `flex recycle` command.
+   _Deprecated_ Shows the job status of a deploy/recycle command. If you don't specify an `id`, the command returns the status of the most recent `flex deploy` or `flex recycle` command. _Deprecation note: For backward compatibility, the command always returns COMPLETE when executed without an `id`. Use `flex status` to track the progress of deploy or recycle commands._
```

Comment:
> command always returns COMPLETE when executed without an `id`

maybe, the backend does always return `COMPLETE` but the CLI still checks if there's a previously saved job ID and if not it outputs the help menu + `This project is not configured. Use 'kinvey flex init' to get started. Alternatively, use positional arguments: id.`

Reply:
The deprecation method is the id. It is the value of the _id field, returned by the server.

Reply:
The rewording is out.